src/Tools/dotnet-dev-certs/src/Program.cs

```diff
@@ -292,24 +289,32 @@ private static int CheckHttpsCertificate(CommandOption trust, IReporter reporter
 
             if (trust != null && trust.HasValue())
             {
-                var store = RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? StoreName.My : StoreName.Root;
-                var trustedCertificates = certificateManager.ListCertificates(store, StoreLocation.CurrentUser, isValid: true);
-                if (!certificates.Any(c => certificateManager.IsTrusted(c)))
+                if(!RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
                 {
-                    reporter.Output($@"The following certificates were found, but none of them is trusted:
-{string.Join(Environment.NewLine, certificates.Select(c => $"{c.Subject} - {c.Thumbprint}"))}");
-                    return ErrorCertificateNotTrusted;
+                    var store = RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? StoreName.My : StoreName.Root;
+                    var trustedCertificates = certificateManager.ListCertificates(store, StoreLocation.CurrentUser, isValid: true);
+                    if (!certificates.Any(c => certificateManager.IsTrusted(c)))
+                    {
+                        reporter.Output($@"The following certificates were found, but none of them is trusted:
+    {string.Join(Environment.NewLine, certificates.Select(c => $"{c.Subject} - {c.Thumbprint}"))}");
+                        return ErrorCertificateNotTrusted;
+                    }
+                    else
+                    {
+                        reporter.Output("A trusted certificate was found.");
+                    }
                 }
                 else
                 {
-                    reporter.Output("A trusted certificate was found.");
+                    reporter.Warn("Checking the HTTPS development certificate trust status was requested. We don't support checking whether the certificate is trusted or not on Linux distributions automatically. " +
```

Comment:
Tone? The text shouldn't use `we`.
```suggestion
                    reporter.Warn("Checking the HTTPS development certificate trust status was requested. Checking whether the certificate is trusted or not is not supported on Linux distributions." +
```
Same for the other texts.